app/views/pages/terms_conditions.html.erb

```diff
@@ -0,0 +1,181 @@
+    <h1 class='govuk-heading-xl'>
+      Terms and Conditions
+    </h1>
+
+    <h2 class='govuk-heading-l'>
+      All Users
+    </h2>
+
+    <h3 class='govuk-heading-m'>
+      Using the website
+    </h3>
+
+    <p class="govuk-body">
+      Please read these Terms of Use (“General Terms”) carefully before using this Teachers: claim back your student
+      loan repayments website (the “Service”).
+    </p>
+
+    <p class="govuk-body">
+      This Service is maintained for your personal use and viewing. Access and use by you of this Service constitutes
+      your acceptance of these General Terms. This takes effect from the date on which you first use this website. If
+      you do not agree to these terms, you must not use this website.
+    </p>
+
+    <p class="govuk-body">
+      If we change these General Terms we will post the revised document here with an updated effective date. If we
+      make significant changes to these terms, we may notify you by other means such as sending an email or posting a
+      notice on our home page.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Information About Us
+    </h3>
+
+    <p class="govuk-body">
+      This Service is operated by [the Department for Education] (“<strong>we</strong>”, “<strong>our</strong>”, or
+      “<strong>us</strong>”). We are a central government department and have our registered office at
+      [REGISTERED OFFICE ADDRESS].
+    </p>
+
+    <p class="govuk-body">
+      You can contact us using the following email address:
+      <a href='mailto: [EMAIL ADDRESS]' class='govuk-link'>[EMAIL ADDRESS]</a>
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Access to the Site
+    </h3>
+
+    <p class="govuk-body">
+      We shall not be liable if for any reason the Site is unavailable at any time or for any period. From time to time,
+      we may restrict access to all or some parts of the Site to users who have registered with us.
+    </p>
+
+    <p class="govuk-body">
+      If you choose, or are provided with, a user identification code, password or any other piece of information as
+      part of our security procedures, you must treat such information as confidential, and you must not disclose it to
+      any third party.
+    </p>
+
+    <p class="govuk-body">
+      We reserve the right to restrict or deny you access to all or some part of the Site if in our opinion, you have
+      failed to comply with these General Terms.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Hyperlinking to the Teachers: claim back your student loan repayments website
+    </h3>
+
+    <p class="govuk-body">
+      We do not object to you linking directly to pages on this site and you do not need to ask permission to do so.
+      However, we do not permit our pages to be loaded into frames on your site. The Teachers: claim back your student
+      loan repayments pages must be displayed in the user's entire browser window.
```

Comment:
I think we don't want people linking to pages in the middle of a claim flow. Not sure if we need to be explicit about that, though.

Reply:
Ok will amend it to say linking to the beginning of the service